sklearn/model_selection/partition.py

```diff
@@ -0,0 +1,1129 @@
+"""
+The :mod:`sklearn.model_selection.partition` module includes classes and
```

Comment:
Those second-level module names are mostly for internal organisation, so
I'm quite ambivalent

On 7 June 2015 at 18:56, Raghav R V <notifications@github.com> wrote:

> In sklearn/model_selection/partition.py
> <https://github.com/scikit-learn/scikit-learn/pull/4294#discussion_r31875049>
> :
>
> > @@ -0,0 +1,1129 @@
> > +"""
> > +The :mod:`sklearn.model_selection.partition` module includes classes and
>
> I'll go ahead and do so since it was previously accepted... Please let me
> know if I should keep it as validation though :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/4294/files#r31875049>.
>